lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnDriverRestartConfiguration.java

```diff
@@ -34,10 +38,16 @@
 @Provided
 public final class YarnDriverRestartConfiguration extends ConfigurationModuleBuilder {
   /**
+   * The Evaluator Preserver implementation used for YARN. Defaults to DFSEvalutorPreserver.
```

Comment:
There is not `@DefaultImplementation` annotation on the `EvaluatorPreserver` interface. How do we enforce the default?